README.md

```diff
@@ -9,33 +9,50 @@
 
 <p align="center"><img src="https://woo.com/wp-content/themes/woo/images/logo-woocommerce-bubble.svg" alt="WooCommerce" style="width:100px;height:auto;"></p>
 
-## Documentation
-
-For more detailed information on QIT and how to use it, refer to the [documentation](https://qit.woo.com/docs/).
-
 ## Quality Insights Toolkit (QIT)
 
 QIT is a testing platform for WordPress Plugins and Themes developed by WooCommerce that allows developers to run a series of managed tests out-of-the-box. We are currently in closed beta operating only in the Woo Marketplace.
 
-- **Comprehensive Testing Suite**: Includes various tests to ensure thorough quality checks.
-  - [Woo E2E Test](https://qit.woo.com/docs/test-types/woo-e2e)
-  - [Woo API Test](https://qit.woo.com/docs/test-types/woo-api)
-  - [Activation Test](https://qit.woo.com/docs/test-types/activation)
-  - [Security Test](https://qit.woo.com/docs/test-types/security)
-  - [PHPStan Test](https://qit.woo.com/docs/test-types/phpstan)
-  - [PHP Compatibility Test](https://qit.woo.com/docs/test-types/phpcompatibility)
-  - [Malware Test](https://qit.woo.com/docs/test-types/malware)
-  - _Performance Tests (Work-in-progress)_
-  - _Custom E2E Tests (Work-in-progress)_
+#### Managed Tests
+
+Run tests out-of-the box with zero configuration.
+
+- [Woo E2E Test](https://qit.woo.com/docs/test-types/woo-e2e)
+- [Woo API Test](https://qit.woo.com/docs/test-types/woo-api)
+- [Activation Test](https://qit.woo.com/docs/test-types/activation)
+- [Security Test](https://qit.woo.com/docs/test-types/security)
+- [PHPStan Test](https://qit.woo.com/docs/test-types/phpstan)
+- [PHP Compatibility Test](https://qit.woo.com/docs/test-types/phpcompatibility)
+- [Malware Test](https://qit.woo.com/docs/test-types/malware)
+- _Performance Tests (Work-in-progress)_
+
+#### **Custom E2E Tests** _(Coming Soon)_
+
+Write your own E2E tests and share them with the community, or leverage pre-built ones to effortlessly increase your test coverage. Run compatibility tests with other plugins, and, with a little bit of configuration, create a matrix to run the same tests using various versions of PHP, WP and Woo, for increased coverage. [Read more about Custom E2E Tests.](https://qit.woo.com/docs/custom-tests/introduction)
```

Comment:
Minor nit:

```suggestion
Write your own E2E tests and share them with the community, or leverage pre-built ones to effortlessly increase your test coverage. Run compatibility tests with other plugins, and, with a little bit of configuration, create a matrix to run the same tests using various versions of PHP, WP and Woo, for increased coverage. [Read more about Custom E2E Tests](https://qit.woo.com/docs/custom-tests/introduction).
```

Should we mention anything about Playwright here, something like:

```suggestion
Write your own E2E tests leveraging [Playwright](https://playwright.dev/) and share them with the community, or leverage pre-built ones to effortlessly increase your test coverage. Run compatibility tests with other plugins, and, with a little bit of configuration, create a matrix to run the same tests using various versions of PHP, WP and Woo, for increased coverage. [Read more about Custom E2E Tests.](https://qit.woo.com/docs/custom-tests/introduction)
```

Reply:
I've added the framework, and also did some other tweaks.